finviz/screener.py

```diff
@@ -2,7 +2,7 @@
 from finviz.helper_functions.error_handling import NoResults, InvalidTableType
 from finviz.helper_functions.save_data import export_to_db, export_to_csv
 from finviz.helper_functions.display_functions import create_table_string
-from urllib.parse import urlencode
+from urllib.parse import urlencode, urlparse, parse_qs as urlparse_qs
```

Comment:
You could use `http_request_get()` from `finviz.helper_functions.request_functions.`, instead of importing other parsing functions (keeps the imports to a minimum and does not add unneccessary methods)

Reply:
`http_request_get()` is to "Sends a GET HTTP request to a website and returns its HTML content and full url address.". `urlparse` is to split an url into its components. `parse_qs` is to split the query section into key value pairs. Those are not the same.